### DISCUSSION TITLE
Friction/Roughness factor for walls

### DISCUSSION THREAD

**Discussion by mdariz11** [firemodels/fds#13672] [General]
Hello,
Is there any option to include friction or roughness factor in fds for wall. 

Thanks in advance

**Comment by mcgratta** [comment]
LMGTFY -- How do I apply roughness in FDS Fire Dynamics Simulator?"

or, given that you have spelled "roughness" correctly, why not just search the FDS User's Guide?

**Comment by mcgratta** [comment]
All snarkiness aside, this forum is not a substitute for searching the User's Guide.
